packages/server/mailing/mailing.schema.js

```diff
@@ -111,17 +111,17 @@ MailingSchema.post('find', function () {
   }
   this._startTime = null;
 });
-
-MailingSchema.plugin(mongooseHidden, {
-  hidden: {
-    __v: true,
-    _company: true,
-    _wireframe: true,
-    wireframe: true,
-    _user: true,
-    author: true,
-  },
-});
+MailingSchema.plugin(mongoosePaginate);
+// MailingSchema.plugin(mongooseHidden, {
+//   hidden: {
+//     __v: true,
+//     _company: true,
+//     _wireframe: true,
+//     wireframe: true,
+//     _user: true,
+//     author: true,
+//   },
+// });
```

Comment:
Why is it commented? In my opinion (and based on my understanding of those lines), we should keep them because they hide from the DB results those fields (https://github.com/mblarsen/mongoose-hidden)

If it's not relevant anymore, remove those lines instead of commenting them ;)

Reply:
Yeah it's not relevent any more because we use projection in our query ;)